src/Source/OrientedImageSource.js

```diff
@@ -0,0 +1,44 @@
+import Source from 'Source/Source';
+
+/**
+ * @classdesc OrientedImageSource is a specific source used to load oriented images.
+ * @extends Source
+ */
+class OrientedImageSource extends Source {
+    /**
+     * @constructor
+     * @param { Object } source - Configuration object
+     * @param { string } source.url - Url for all the textures.
+     * This Url must contains {sensorId} and {cameraId}, and these pattern will be replaced be the {@link OrientedImageProvider} o find the good texture for each camera for each panoramic.
+     */
+    constructor(source) {
+        source.format = source.format || 'json';
+        super(source);
+        this.protocol = 'orientedImage';
```

Comment:
`this.protocol` is from source could you use `this.isOrientedImageSource = true`

Reply:
It's seem to be unused, so I've remove this protocol line.

Reply:
but we must add `this.isOrientedImageSource = true`

Reply:
ok